### DISCUSSION TITLE
Using [addonify_compare_products] shortcode

### DISCUSSION THREAD

**Discussion by amir36** [addonify/addonify-compare-products#227] [Q&A]
Hi everyone,

Kudos to developer for such a fantastic plugin.

I do not know how to use the shortcode and the parameters that can be embed. The docs do not cover this. I am looking to create a shortcode for comparison of 3 specific products. 

Thank you for your support.

Amir

**Comment by h1dd3nsn1p3r** [comment]
Hi @amir36 

Thank you very much. 

This is not how  `[addonify_compare_products]` shortcode works. You won't be able to add 3 products, **save** the compare table and then share the **public link** for now. But yes this sounds like a cool feature to have. 

Since Addonify Compare doesn't have this feature yet, you may create a static page using page builders like Elementor and share the link. I hope this makes sense. 

Thanks,
Anuj
